build.gradle

```diff
@@ -16,6 +16,17 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-mongodb'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	// https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-validation
+	implementation group: 'org.springframework.boot', name: 'spring-boot-starter-validation', version: '3.0.1'
+
```

Comment:
Remove all empty lines like this from all files. Can keep one empty line for code readability, but multiple empty lines are not recommended

Reply:
Removed